src/builder/arg_predicate.rs

```diff
@@ -4,6 +4,7 @@ use crate::builder::OsStr;
 ///
 /// These do not apply to [`ValueSource::DefaultValue`][crate::parser::ValueSource::DefaultValue]
 #[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "unstable-v4", non_exhaustive)]
```

Comment:
Shouldn't this be "**unstable-v5**"?

Reply:
Thanks for catching that.  I guess I've written unstable-v4 quite a lot...